scripts/opdracht-1.js

```diff
@@ -0,0 +1,64 @@
+console.log(inventory);
+
+//---------------------------------------------
+// Opdracht 1a
+
+// let tvTypes = [];
+// for (let i = 0; i < inventory.length; i++) {
+//   tvTypes.push(inventory[i].type);
+// }
+
+const tvTypes = inventory.map((item) => {
+  return item.type;
+})
+
+console.log(tvTypes);
+
+//---------------------------------------------
+// Opdracht 1b
+
+// let soldOut = [];
+//
+// for (let i = 0; i < inventory.length; i++) {
+//   if (inventory[i].originalStock - inventory[i].sold === 0) {
+//     soldOut.push(inventory[i]);
+//   }
+// }
+
+function filterSoldOut(inventory) {
+  return  inventory.filter((item) => {
+    return item.originalStock - item.sold === 0;
```

Comment:
Goed hoor, maar het kan nog iets korter 😉 
```suggestion
    return item.originalStock === item.sold;
```